src/main/webapp/app/entities/student-exam.model.ts

```diff
@@ -22,6 +22,7 @@ export class StudentExam implements BaseEntity {
 
     // helper attribute
     public ended?: boolean;
+    public numberOfExamSessions = 0; // default value
```

Comment:
I would omit that comment
```suggestion
    public numberOfExamSessions = 0;
```